### DISCUSSION TITLE
Resources not found??

### DISCUSSION THREAD

**Discussion by silicahd** [filamentphp/filament#853] [Help]
### Package

filament/filament

### Package Version

2.0

### Laravel Version

8.65

### Livewire Version

2.8

### PHP Version

8.0

### Bug description

After installing and navigating /admin the page does not look correct. When looking at the source files should be located /admin/assets/app.css   ??? I do not have this folder generated. Maybe the issue is Tailwind 3 ?? Installed Tailwind 3 before. Than just ran the install commands in the documentation https://filamentadmin.com/docs/2.x/admin/installation  what am i missing? 

### Steps to reproduce

Install Laravel 
Install Tailwind 3
Install Filament

**Comment by camerontucker** [comment]
Inspecting my browser traffic showed that app.css was trying for http://. Just had to force https.
